src/OrleansRuntime/Messaging/InboundMessageQueue.cs

```diff
@@ -16,22 +17,24 @@ public int Count
             get
             {
                 int n = 0;
-                foreach (var queue in messageQueues)
-                    n += queue.Count;
-                
+                // currently doesn't being used
```

Comment:
"currently doesn't being used" :  and that is a good reason to retrun 0 instead?